docs/training/apis-howto/api-pytorch-trainer-ug.rst

```diff
@@ -0,0 +1,354 @@
+.. _pytorch-trainer-guide:
+
+#####################
+ PyTorch Trainer API
+#####################
+
+This guide will help you get up and running with the PyTorch Trainer API.
+
++-----------------------------------------------------------------------------+
+| Visit the API reference                                                     |
++=============================================================================+
+| :doc:`/reference/reference-training/training/api-pytorch-trainer-reference` |
++-----------------------------------------------------------------------------+
+
+With the PyTorch Trainer API, you can implement and iterate on model training code locally before
+running on cluster. When you are satisfied with your model code, you configure and submit the code
+on cluster.
+
+The PyTorch Trainer API lets you do the following:
+
+-  Work locally, iterating on your model code.
+-  Debug models in your favorite debug environment (e.g., directly on your machine, IDE, or Jupyter
+   notebook).
+-  Run training scripts without needing to use an experiment configuration (YAML) file.
+-  Load previous saved checkpoints directly into your model.
+
+************
+ Objectives
+************
+
+After completing the steps in this guide, you will be able to do the following:
+
+-  Define a PyTorch Trial
+-  Initialize the PyTorch Trial and the Trainer
+-  Run Your Training Script Locally
+-  Submit Your Trial for Training On Cluster
+-  Load checkpoints
+
+***************
+ Prerequisites
+***************
+
+-  Access to a Determined cluster. If you have not yet installed Determined, refer to the
+   :ref:`installation instructions <install-cluster>`.
+
+-  The Determined CLI should be installed on your local machine. For installation instructions,
+   visit :ref:`Commands and Shells: Installation <install-cli>`. After installing the CLI, configure
+   it to connect to your Determined cluster by setting the ``DET_MASTER`` environment variable to
+   the hostname or IP address where Determined is running.
+
+********************************
+ Step 1: Define a PyTorch Trial
+********************************
+
+Start by defining a PyTorchTrial by instantiating the `Trial` and `TrialContext` objects. When using
+the PyTorch Trainer API, you do not have to initialize and wrap models and optimizers inside of the
+`Trial.__init__` method. If desired, pass a wrapped model to `Trial.__init__`.
+
+..
+   code::python
```

Comment:
This linebreak between `..` and `code::python` turns this block into a block comment rather than a code block.  Nothing renders on the built page.

Reply:
it does this when i forget to add the space between code:: and python
resolved